normal.tex

```diff
@@ -97,7 +97,7 @@ \section{Before Starting the Engine}
   \item Alternate air -- OFF (Emergency use only)
   \item Brakes -- Test
   \item Avionics -- OFF
-  \item Flaps -- RETRACT IN STAGES ($20^\circ, 10^\circ, 0^\circ)$
+  \item Flaps -- RETRACT IN STAGES ($20^\circ, 10^\circ, 0^\circ$)
```

Comment:
Our flap angles are closer to 0, 15, 30 and 40+ degrees

Reply:
Fixed